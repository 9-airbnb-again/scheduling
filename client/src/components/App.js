import React, { useState } from 'react';
import Checkout from './Checkout';

export default function App() {
  const [accommodation, setAccommodation] = useState(null);

  fetch('/api/v1/listings').then((res) => {
    return res.json();
  }).then((accommodations) => {
    setAccommodation(accommodations[12]);
  });

  return (
    <div className="App">
      { accommodation && <Checkout accommodation={accommodation} /> }
    </div>
  );
}
