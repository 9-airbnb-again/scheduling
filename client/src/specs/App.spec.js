import React from "react";
import { shallow } from "enzyme";
import App from "../components/App.js";
import "isomorphic-fetch";

describe("App", () => {
  test("App component should mount", () => {
    const wrap = shallow(<App />);
    expect(wrap).toMatchSnapshot();
  });
});
