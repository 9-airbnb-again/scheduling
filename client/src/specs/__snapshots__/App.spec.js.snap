// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App component should mount 1`] = `ShallowWrapper {}`;
